src/app.rs

```diff
@@ -14,10 +14,10 @@ Project home page: https://github.com/BurntSushi/ripgrep
 Use -h for short descriptions and --help for more details.";
 
 const USAGE: &'static str = "
-    rg [OPTIONS] <pattern> [<path> ...]
-    rg [OPTIONS] [-e PATTERN | -f FILE ]... [<path> ...]
-    rg [OPTIONS] --files [<path> ...]
-    rg [OPTIONS] --type-list";
+    rg [options] PATTERN [path ...]
+    rg [options] (-e PATTERN | -f FILE) [path ...]
```

Comment:
Oh I see, because -e and -f can be defined multiple times... As both -e and -f can also be used simultaneously, what about:

    rg [options] [-e pattern ...] [-f file ...] [path ...]

?